inst/data_scripts/update_mh.R

```diff
@@ -0,0 +1,42 @@
+# Add new variables to MH
+library(dplyr)
+library(tidyselect)
+library(labelled)
+library(admiral)
+library(metatools)
+library(admiral.test)
+
+data("raw_mh")
+
+# Convert blank to NA
+mh <- convert_blanks_to_na(raw_mh)
+set.seed(1)
```

Comment:
Can you put a comment on why we need to set the seed?

Reply:
Updated